src/main/resources/templates/snowflake/Makefile.schema.ssp

```diff
@@ -14,11 +14,24 @@ See the License for the specific language governing permissions and
 limitations under the License.
 --%>
 <%@ val tables: Seq[io.phdata.streamliner.schemadefiner.model.TableDefinition] %>
-<%
-	val firstRunTables = tables.map(t => s"first-run-${t.destinationName}").mkString(" ")
-	val runTables = tables.map(t => s"run-${t.destinationName}").mkString(" ")
-	val cleanTables = tables.map(t => s"clean-${t.destinationName}").mkString(" ")
-%>
+<%@ val tableDiffs: Seq[io.phdata.streamliner.schemadefiner.model.TableDiff] %>
+#if (!tableDiffs.isEmpty)
+    <%
+        val runTables = tableDiffs.map(t => s"run-${t.destinationName}").mkString(" ")
+    %>
+run-all: ${runTables}
+
+#for (table <- tableDiffs)
+run-${table.destinationName}:
+	$(MAKE) run -C ${table.destinationName}
+
+#end
+#elseif (tables != null)
+    <%
```

Comment:
I think on most runs, both tableDiffs AND tables are going to have data. Should this be in a seperate if, not an else-if?

Reply:
This is what I am thinking 
1. initial script to create table  -> conifg present configDiff absent
2. evolve schema to add new columns -> config present configDiff present 

For 2nd case, currently only evolve schema scripts are generated. Initial scripts are already generated during first run of script command. If we add separate if here then again whole initial scripts  and evolve schema scripts will be generated.